st2common/bin/st2ctl

```diff
@@ -57,16 +57,27 @@ function validate_in_components() {
 
 function st2start() {
   for COM in ${COMPONENTS}; do
-    service ${COM} start
+    service_manager ${COM} start
   done
 }
 
 function st2stop() {
   for COM in ${COMPONENTS}; do
-    service ${COM} stop
+    service_manager ${COM} stop
   done
 }
 
+function service_manager() {
+  if command -v systemctl 2>/dev/null; then
+    systemctl "${2}" "${1}"
```

Comment:
So that CentOS7 screenshot is from Docker?
See my comments there: https://github.com/StackStorm/st2/pull/2367/files#r49769025